sources/Interop/Kernel32/um/synchapi/Kernel32.Manual.cs

```diff
@@ -7,21 +7,21 @@ namespace TerraFX.Interop
 {
     public static unsafe partial class Kernel32
     {
-        public const uint INIT_ONCE_CHECK_ONLY = RTL_RUN_ONCE_CHECK_ONLY;
+        public const uint INIT_ONCE_CHECK_ONLY = Windows.RTL_RUN_ONCE_CHECK_ONLY;
```

Comment:
These have been `using static TerraFX.Interop.Windows` thus far to help make comparing against native code easier.

Reply:
Fixed.